### PR TITLE
TEC-3992 Sharing Block Re-Skin

### PR DESCRIPTION
**Pull request by EugeneKyale**
Artifact : https://www.loom.com/share/7df920c8f69f41d185d684b753e94b67
Ticket : https://theeventscalendar.atlassian.net/browse/TEC-3992

There is a related PR for this in Common here : https://github.com/the-events-calendar/tribe-common/pull/1641

### CODE REVIEW

File: src/styles/event-links/frontend.pcss [hunk 1]
```diff
@@ -15,23 +16,47 @@
...
+
+/* Genesis */
+.theme-genesis .tribe-block__events-link .tribe-block__btn--link a {
+	font-size: 1.65rem;
+}
+
+/* Avada */
+.theme-Avada .tribe-block__events-link .tribe-block__btn--link a::before {
+	content: '';
+}
```
Comment (paulmskim) [inline src/styles/event-links/frontend.pcss:62]: same note on avada here.

File: src/styles/event-links/frontend.pcss [hunk 2]
```diff
@@ -15,23 +16,47 @@
...
-			font-weight: bold;
 			font-size: 1rem;
 			justify-content: center;
-			margin: 10px 15px 10px 0;
-			min-height: 38px;
-			padding: 0 17px 0 14px;
+			margin: 10px 30px 10px 0;
+			position: relative;
+
+			&::before {
```
Comment (paulmskim) [inline src/styles/event-links/frontend.pcss:26]:
```suggestion
			&:before {
```

**Review by paulmskim — CHANGES_REQUESTED** [review]
one small fix, also let's make sure we touch base with Avada.

**Comment by EugeneKyale** [comment]
I'm closing this PR because we're using a different approach for this task. 
Reference : https://github.com/the-events-calendar/the-events-calendar/pull/3627

**Pull request closed without merging**